### ISSUE TITLE
Addition of ConfirmPasswordController.php breaking change

### ISSUE THREAD

**Issue by danabrey** [laravel/framework#30382]
- Laravel Version: 6.2
- PHP Version: 7.3
- Database Driver & Version:

### Description:

[This issue](https://github.com/laravel/framework/issues/30227) involving missing ConfirmPasswordController occurs when upgrading from Laravel 6.0 to 6.2. It's not a backwards-compatible new feature, and requires adding new files to the application codebase to avoid errors. 

Doesn't this break the idea of semantic versioning? Regardless, I am unable to find an 'upgrade guide' for 6.0 to 6.1 or 6.1 to 6.2 - maybe I'm missing them? Guides like this shouldn't be as necessary for a project that respects semver, but if changes like this are occurring they become crucial.

### Steps To Reproduce:

- Create a Laravel 6.0 project
- Upgrade to Laravel 6.2

**Comment by alec-joy** [comment]
Where are you getting this error message? We're on Laravel 6.3, upgraded from 5.8 (all the way back to 5.4 actually) and although we don't have this controller, we don't get this error anywhere

**Comment by devcircus** [comment]
Since the 6.3 release I’ve only seen 2-3 devs mention problems. Has to be something else going on. The repo would be flooded with issues if this was a problem.

**Comment by alec-joy** [comment]
Okay I think I see the issue, our app doesn't use the Auth::routes(); helper as we have our own auth logic. If you use Auth::routes() I do get this error. That's the extra step.

**Comment by devcircus** [comment]
Interesting. If true, it’s something that should not have been added if Laravel is serious about semver.

**Comment by Tantriss** [comment]
Hi !
Same problem here, got the error after upgrading from 6.1 to 6.3 and it was because I use 
```
Auth::routes([
    'reset'    => false,
    'verify'   => false,
    'register' => false,
]);
```
in my web.php route file.
I had to add `confirm => false` to fix it.

**Comment by browner12** [comment]
Let's get rid of `Auth::routes()` and force users to be explicit about their routes.

**Comment by ajoy39** [comment]
That might be an option long term but it would be a breaking change and 6.x is an LTS version of laravel, so it wouldn't help solve the core issue. There either needs to be a fallback in the app or a conditional around registering those routes if the file doesn't exist

**Comment by netpok** [comment]
Actually this only breaks the route:list command and the newly introduced endpoints as far as I know, I tried route:cache, it's not affected.

That being I don't think this feature should have been added enabled by default, actually I think this should be disabled by default, just like the verify option.

Sadly I think disabling it after a week would probably cause more breaks in production instances than leaving it.

This definitely should be considered next time.
